xml/System.Windows.Forms/AccessibleEvents.xml

```diff
@@ -735,6 +764,15 @@
       <MemberValue>7</MemberValue>
       <Docs>
         <summary>A shortcut menu was closed. The system sends this event for standard menus. Servers send this event for custom menus. When a shortcut menu is closed, the client receives this message followed almost immediately by the <see langword="SystemMenuEnd" /> event. The system does not send the event consistently.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Note that for a call to TrackPopupMenu(), a client will see `EVENT_SYSTEM_MENUSTART` followed almost immediately by `EVENT_SYSTEM_MENUPOPUPSTART` for the popup being shown.
+
+          ]]></format>
+        </remarks>
```

Comment:
Is this a remark only relevant for developers reading the source code? `TrackPopupMenu()` seems to be [a Windows method](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-trackpopupmenu), that's why I ask.

If you think that is the case, we can remove it:

```suggestion
        <remarks>To be added.</remarks>
```

Reply:
@RussKie You added this comment a while back, should we leave it as is?